test/integration/scheduler/workload_controller_test.go

```diff
@@ -467,6 +470,131 @@ var _ = ginkgo.Describe("Workload controller with scheduler", func() {
 		})
 	})
 
+	ginkgo.When("Resource transformations are applied", func() {
+		ginkgo.BeforeEach(func() {
+			gomega.Expect(k8sClient.Create(ctx, onDemandFlavor)).To(gomega.Succeed())
+			clusterQueue = testing.MakeClusterQueue("clusterqueue").
+				ResourceGroup(*testing.MakeFlavorQuotas(onDemandFlavor.Name).
+					Resource(corev1.ResourceCPU, "5", "5").Obj()).
+				Cohort("cohort").
+				Obj()
+			gomega.Expect(k8sClient.Create(ctx, clusterQueue)).To(gomega.Succeed())
+			localQueue = testing.MakeLocalQueue("queue", ns.Name).ClusterQueue(clusterQueue.Name).Obj()
+		})
+		ginkgo.AfterEach(func() {
+			gomega.Expect(util.DeleteNamespace(ctx, k8sClient, ns)).To(gomega.Succeed())
+			util.ExpectObjectToBeDeleted(ctx, k8sClient, clusterQueue, true)
+			util.ExpectObjectToBeDeleted(ctx, k8sClient, onDemandFlavor, true)
+		})
+
+		ginkgo.It("The transformed resources should be used as request values", func() {
```

Comment:
In separate `ginkgo.It`, could you verify if the Workload object does not have the `.status.resourceRequests` and the transformation does not reflected?

I guess that we need to prepare 2 ginkgo nodes:

1. Disable both ConfigurableResourceTransformations and WorkloadResourceRequestsSummary, and enqueue the Workload with pseudoCPU. And then, we verify if the admission does not have transformed parameters and the .status.resourceRequests does not exist.
2. Disable ConfigurableResourceTransformations, and enqueue the Workload with pseudoCPU. And then, we verify if the admission and resourceRequests do not have transformed parameters.